### PR TITLE
Update KF GCP deployment instructions to use the new manifest repo

### PR DESCRIPTION
**Pull request by chensun**

### CODE REVIEW

File: content/en/docs/distributions/gke/deploy/management-setup.md [hunk 1]
```diff
@@ -31,41 +31,41 @@ to manage Google Cloud infrastructure using GitOps.
     Note: Starting from Kubeflow 1.4, it requires `kpt v1.0.0-beta.6` or above to operate in `kubeflow/gcp-blueprints` repository. gcloud hasn't caught up with this kpt version yet, [install kpt](https://kpt.dev/installation/) separately from https://github.com/GoogleContainerTools/kpt/tags for now. Note that kpt requires docker to be installed.
 
 
-### Fetch kubeflow/gcp-blueprints package
+### Fetch GoogleCloudPlatform/kubeflow-distributio package
```
Comment (gkcalat) [inline content/en/docs/distributions/gke/deploy/management-setup.md:34]: This should be:
```
### Fetch GoogleCloudPlatform/kubeflow-distribution package
```

**Review by gkcalat — CHANGES_REQUESTED** [review]
Thank you @chensun!
There is one little typo...

File: content/en/docs/distributions/gke/deploy/management-setup.md (hunk 1 above)
Comment (chensun) [reply content/en/docs/distributions/gke/deploy/management-setup.md]: Fixed

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thanks for the update!

**Pull request merged**